BlogTemplate/Pages/Drafts.cshtml.cs

```diff
@@ -21,10 +21,11 @@ public DraftsModel(BlogDataStore dataStore)
         }
 
         public IEnumerable<Post> Posts { get; set; }
-
+
+        [ValidateAntiForgeryToken]
```

Comment:
I believe this attribute is only needed on Post methods (i.e. when a form is being submitted)